assets/lambda/code/scan/lambda.py

```diff
@@ -104,15 +104,24 @@ def lambda_handler(event, context):
 
 
 def set_status(bucket, key, status):
+    """Get previous tags and append scan-status tag"""
+    old_tags = {}
+    try:
+        response = s3_client.get_object_tagging(Bucket=bucket,Key=key)
+        old_tags = {i['Key']: i['Value'] for i in response['TagSet']}
+    except Exception as e:
+        logger.debug("No tags")
+
```

Comment:
```suggestion
    except botocore.exceptions.ClientError as e:
        logger.debug(e.response["Error"]["Message"])
```
If the command is run on a object with no tags, the response returns `'TagSet': []` instead of an error. We can check for any other client errors here instead.